REQUIRE

```diff
@@ -1,6 +1,6 @@
 julia 0.4
 Distributions 0.6+
 HDF5
-KernelDensity
-Grid
+KernelDensity 0.2.0 0.3.0
+Grid 0.4.2
```

Comment:
Is there an actual reason to require `Grid 0.4.2`
We should specify the earliest compatible version, not require the latest, if we don't have to